Resources/Prototypes/DeltaV/Catalog/Jukebox/Standard.yml

```diff
@@ -0,0 +1,92 @@
+# sorted alphabetically based on filenames in the folder Resources/Audio/DeltaV/Jukebox
+# keep it ordered or I'll stab you
+
+- type: jukebox
+  id: ADiffReal
+  name: Andreas Viklund - A.D.R (Lagoona rmx)
+  path:
+    path: /Audio/DeltaV/Jukebox/a_different_reality_lagoona_remix.xm-MONO.ogg
+
+- type: jukebox
+  id: AggAss
+  name: melcom - Aggravated Assault
+  path:
+    path: /Audio/DeltaV/Jukebox/aggravated.it-MONO.ogg
+
+- type: jukebox
+  id: AutEqu
+  name: lemonade - Autumnal Equinox
+  path:
+    path: /Audio/DeltaV/Jukebox/autumnal_equinox.xm-MONO.ogg
+
+- type: jukebox
+  id: DosHighUmb
+  name: MASTER BOOT RECORD - DOS=HIGH, UMB
+  path:
+    path: /Audio/DeltaV/Jukebox/DOS=HIGH,_UMB.ogg
+
+- type: jukebox
+  id: DrozAlone
+  name: Drozerix - Alone
+  path:
+    path: /Audio/DeltaV/Jukebox/drozerix_-_alone.xm-MONO.ogg
+
+- type: jukebox
+  id: DrozLeisure
+  name: Drozerix - Leisurely Voice
+  path:
+    path: /Audio/DeltaV/Jukebox/drozerix_-_leisurely_voice.xm-MONO.ogg
+
+- type: jukebox
+  id: FemtMurder
+  name: Femtanyl feat. takihasdied - MURDER EVERY 1 U KNOW
+  path:
+    path: /Audio/DeltaV/Jukebox/femtanyl_-_MURDER_EVERY_1_U_KNOW_feat._takihasdied_MONO.ogg
+
+- type: jukebox
+  id: KCHaxors
+  name: Karl Casey @ White Bat Audio - Hackers
+  path:
+    path: /Audio/DeltaV/Jukebox/hackers-MONO.ogg
+
+- type: jukebox
+  id: IanMarhaba
+  name: Ian Alex Mac. - Marhaba
+  path:
+    path: /Audio/DeltaV/Jukebox/marhaba-MONO.ogg
+
+- type: jukebox
+  id: PTMinute
+  name: Patricia Taxxon - Minute
+  path:
+    path: /Audio/DeltaV/Jukebox/Patricia_Taxxon_-_Minute_-_MONO.ogg
+
+- type: jukebox
+  id: NymphsForest
+  name: Psirius - Nymphs of the Forest
+  path:
+    path: /Audio/DeltaV/Jukebox/psirius_-_nymphs_of_the_forest.mptm-MONO.ogg
+
+- type: jukebox
+  id: GhirScratch
+  name: ghirardelli7 - Scratch Post
+  path:
+    path: /Audio/DeltaV/Jukebox/Scratch_Post_-_OST_MONO.ogg
+
+- type: jukebox
+  id: JukeShiba
+  name: Dot Nigou - Shibamata
+  path:
+    path: /Audio/DeltaV/Jukebox/shibamata-MONO.ogg
+
+- type: jukebox
+  id: SpaceAsshowl
+  name: Chris Remo - Space Asshole
+  path:
+    path: /Audio/DeltaV/Jukebox/space_asshole-MONO.ogg
+
+- type: jukebox
+  id: AmieSuperpos
+  name: Amie Waters - Superposition
+  path:
+    path: /Audio/DeltaV/Jukebox/superposition-MONO.ogg
```

Comment:
```suggestion
    path: /Audio/DeltaV/Jukebox/superposition-MONO.ogg

```
line at the end is good form

Reply:
most files upstreams have lines at the end, it's good form and mentally makes it easy to add onto the PR as there's already an existing line and open space for writing

Reply:
Source: Me.

Reply:
ok